python/sedonadb/tests/functions/test_transforms.py

```diff
@@ -93,3 +93,32 @@ def test_st_crs_sedonadb(eng):
         "SELECT ST_CRS(ST_SetCrs(ST_GeomFromText('POINT (1 1)'), NULL))",
         None,
     )
+
+
+@pytest.mark.parametrize("eng", [SedonaDB, PostGIS])
+@pytest.mark.parametrize(
+    ("geom", "dx", "dy", "expected"),
+    [
+        (None, None, None, None),
```

Comment:
```suggestion
        (None, None, None, None),
        (None, 1.0, 2.0, None),
```
nit: While I don't think it's necessary to test every single combination of nulls, I think it's nice to at least have a case for each arg where *only* that arg is null, so we know that arg on it's own is enough to trigger a null output. There are currently cases for only dx null, only dy null, but not "only geometry null" (which I'd guess is the most likely case).